docs/core/compatibility/sdk/6.0/implicit-namespaces.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Breaking change: Implicit namespaces in C# projects"
+description: Learn about the breaking change in .NET 6 where the .NET SDK implicitly includes some namespaces globally in C# projects.
+ms.date: 07/20/2021
+---
+# Implicit namespaces in C# projects
+
+.NET 6 introduces implicit namespace support for C# projects. To reduce the amount of `using` directives boilerplate in .NET C# project templates, namespaces are implicitly included by utilizing the `global using` feature introduced in C# 10.
+
+The implicit namespace feature is enabled by default for C# projects that target the `net6.0` TFM or higher and use the Microsoft.NET.Sdk, Microsoft.NET.Sdk.Web, or Microsoft.NET.Sdk.Worker SDK. The feature generates a source file that's passed to the compiler and contains `global using` directives for the default namespaces.
```

Comment:
What happens if the TFM is `net6.0` but with lower language version that doesn't support `global using`?